apps/system/test/unit/accessibility_test.js

```diff
@@ -289,4 +313,67 @@ suite('system/Accessibility', function() {
         accessibility.quicknav.element.classList.contains('visible'));
     });
   });
+
+  suite('interaction hints', function() {
+    var screenNode, clock;
+    var realL10n = navigator.mozL10n;
+
+    setup(function() {
+      screenNode = document.createElement('div');
+      screenNode.id = 'screen';
+      document.body.appendChild(screenNode);
+
+      clock = this.sinon.useFakeTimers();
+
+      navigator.mozL10n = MockL10n; // jshint ignore:line
```

Comment:
For whatever reason, jshint wouldn't let me commit the file because it was saying MockL10n was undefined.  I have no idea why it wasn't picking up the 'require' statement earlier in the doc, but if there's something you notice as to why that would be happening, I'd love to know the answer!